ups/drp_stella.table

```diff
@@ -3,6 +3,7 @@ setupRequired(pipe_drivers)
 setupRequired(afw)
 setupRequired(meas_base)
 setupRequired(obs_pfs)
+setupRequired(drp_fstar)
```

Comment:
We need to update our scripts to handle the extra package (currently, the integration test fails). Do we really need as an extra package? As far as I can see, it adds just a few extra python scripts that could just as well live in drp_stella.

Reply:
Yamashita-san agreed that `drp_fstar` can be merged to `drp_stella`. ~Should I make another ticket~ Yamashita-san is making another ticket to merge `drp_fstar` to `drp_stella`. I will make a PR for it before working on this ticket further.